docs/concepts/securing-lokomotive-cluster.md

```diff
@@ -0,0 +1,59 @@
+# Securing Lokomotive clusters
+
+Lokomotive clusters are deployed with security as a top concern.
+
+## Cluster wide Pod Security Policy
+
+A [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) is a
+cluster-level resource that controls security sensitive aspects of the pod specification. The
+PodSecurityPolicy objects define a set of conditions that a pod must run with in order to be
+accepted into the system, as well as defaults for the related fields.
+
+Lokomotive clusters have PodSecurityPolicy (PSP) enabled by default. The cluster comes with two
+default PSPs for general purpose application usage:
+
+  * [restricted](../../assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/psp-restricted.yaml)
+
+    Allowed to all the workloads in all namespaces except `kube-system` namespace. This PSP has the
+    following restrictions:
+
+    * Does not allow pods to be run as root.
+    * Allows only whitelisted volumes.
+    * Allows only whitelisted capabilities.
+    * The linux kernel host namespace sharing is not allowed for any pod using this PSP. The default
+      docker seccomp profiles are used.
+
+    To allow special permissions to workloads, the recommended way is to create a bespoke PSP and
+    allow the workloads’ ServiceAccount to use that PSP. Refer to the
+    [documentation](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) on creating a
+    Pod Security Policy.
+
+  * [privileged](../../assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/psp-privileged.yaml)
+
+    Allowed to workloads in `kube-system` namespace only. This PSP does not restrict any workloads
+    from any required permissions.
+
+## Global network policy (for Packet platform only)
+
+Lokomotive installs Calico’s
+[GlobalNetworkPolicy](https://docs.projectcalico.org/security/calico-network-policy) by default.
+This helps with restricting access to the nodes from outside the cluster.
+
+The policy named `ssh` is worth noting, since there we can define who can ssh into the nodes. To edit
+the IP address list, run following command:
+
+```console
+kubectl edit globalnetworkpolicies ssh
```

Comment:
I don't think we should suggest doing that for updating the policies. All updates should be done via the CLI.

Reply:
We've also mentioned a note saying that the changes done in that manner will be temporary until the next time `lokoctl cluster install` is triggered.

Reply:
I know, but still I think we shouldn't promote such solution. We could do other way around, say that recommended way is editing the configuration and rolling cluster upgrade, but if change is only temporary, it can be done via `kubectl edit`.

Reply:
`lokoctl cluster install` approach has also this benefit, that it's platform-agnostic and hide the implementation details from the user.

Reply:
This is addressed below.